deps.json

```diff
@@ -6,15 +6,15 @@
       "subrepo" : "llvm-mirror/llvm",
       "branch" : "master",
       "subdir" : "third_party/llvm",
-      "commit" : "82f73ee5b37a2a4cc1bdad02bebaaaba71b65400"
+      "commit" : "26882c9d258b62748a7266207513a06990c8decc"
```

Comment:
This gets us in LLVM and Clang 9

Reply:
I'm going to have to make sure that `clspv-opt` still works with this.  Do you know if LLVM 9 removed support for the old pass manager to `opt`?

Reply:
We still use it in clspv, so it works.